src/owui_ext/shared/completion_response.py

```
@@ -0,0 +1,98 @@
+        text = str(body)
+    text = text.strip()
+    if len(text) > _RESPONSE_BODY_PREVIEW_CHARS:
+        text = text[:_RESPONSE_BODY_PREVIEW_CHARS] + "...[truncated]"
+    return text
+
+
+def _extract_json_response_error(response: JSONResponse) -> str:
+    status = getattr(response, "status_code", "unknown")
+    try:
+        error_data = json.loads(bytes(response.body).decode("utf-8"))
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)